guides/v3.4.0/models/index.md

```diff
@@ -1,5 +1,19 @@
-Models are objects that represent the underlying data that your
-application presents to the user. Different apps will have very
+Ember developers have great options for how they handle data from back end APIs. 
+Ember itself works with any type of back end: REST, JSON API, GraphQL, or anything else.
+
+Many developers choose to use Ember Data, a powerful set of tools for formatting requests, serializing responses, and efficiently managing a local cache of data. 
```

Comment:
I will bring this up with the `ember-data` team, but I think it is time we officially settled on how to reference the library in technical writing. Some places we use `Ember Data`, some places `ember-data`, and still other places `EmberData`. I am preferential to using `ember-data` as it matches the package name.

Reply:
We have styled it "Ember Data", "Ember", and "Ember CLI" throughout our resources, unless specifically mentioning the repository/library, like "it can easily be removed by removing the `ember-data` entry in `package.json`." mentioned below.

Reply:
It would be nice to champion a change away from `Ember Data` to `ember-data`. Unfortunately, I too often see users incorrectly confusing/mixing concepts from `Ember` and `Ember Data` and a good deal of it seems to stem from the branding.

Reply:
I have opened an issue for discussion from the learning team perspective, #184.

I think calling it ember-data has some nice benefits. I will leave it as Ember Data, for the scope of this PR, but please let me know what the data team settles on.